packages/wallet/puppeteer/helpers.ts

```diff
@@ -50,15 +50,13 @@ export async function loadWallet(page: puppeteer.Page, messageListener: (message
   const web3JsFile = fs.readFileSync(path.resolve(__dirname, "web3/web3.min.js"), "utf8");
   await page.evaluateOnNewDocument(web3JsFile);
   await page.evaluateOnNewDocument(`window.web3 = new Web3("http://localhost:${port}")`);
-  await page.goto("http://localhost:3055/", {waitUntil: "networkidle0"});
+  await page.goto("http://localhost:3055/", {waitUntil: "load"});
   page.on("pageerror", error => {
     throw error;
   });
   page.on("console", msg => {
     if (msg.type() === "error") {
       throw new Error(`CONSOLE ERROR: ${msg.text()}`);
-    } else {
```

Comment:
Why remove this condition?

Reply:
Because it does this:

```
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
    console.log puppeteer/helpers.ts:61
      CONSOLE LOG: JSHandle@object
```